packages/client/src/views/SysAdmin/Team/user/userProfilie/UserProfile.tsx

```diff
@@ -43,7 +43,7 @@ import { IStoreState } from '@client/store'
 import { GenericErrorToast } from '@client/components/GenericErrorToast'
```

Comment:
This component can now be removed altogether

> Combine UserProfile.tsx and UserAuditList.tsx into a component called UserAudit and deprecate the term UserProfile / UserAuditList.

https://github.com/opencrvs/opencrvs-core/issues/3640